### PR TITLE
bumper, Remove Draft flag from bumper generated PRs

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
Currently PRs opened by the bumper script are
in draft mode. Since the script is stable we
want to move script to issue real PRs.
Removed Draft flag from created PRs

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
/hold
to avoid potential issues, let's wait until #648 is merged and run until we merge this PR

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Comment by RamLavi** [comment]
/hold cancel 

I think the bumper is mature enough to go online.
FYI @phoracek @qinqon

**Pull request merged**
